src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeZoneToStringTransformer.php

```diff
@@ -39,14 +39,14 @@ public function transform($dateTimeZone)
 
         if ($this->multiple) {
             if (!\is_array($dateTimeZone)) {
-                throw new TransformationFailedException('Expected an array.');
+                throw new TransformationFailedException('Expected an array of \DateTimeZone objects.');
```

Comment:
the title of the pr should probably change then.

Reply:
sure :D